CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/importing/ImportGraphFilePlugin.java

```diff
@@ -153,7 +153,7 @@ protected RecordStore query(final RecordStore query, final PluginInteraction int
             LOGGER.log(Level.SEVERE, "Unable to process graph file");
         }
         
-        interaction.setProgress(1, 0, "Completed successfully - added " + result.size() + " entities.", true);
+        interaction.setProgress(1, 0, "Imported " + result.size() + " entity(ies).", true);
```

Comment:
Not sure I like the look of this: "entity(ies)" 
Can you put in a boolean check to show either "entity" or "entities" as appropriate.